app/lib/shared/redis_cache.dart

```diff
@@ -81,13 +81,9 @@ class SimpleMemcache {
   SimpleMemcache(String prefix, Logger logger, Duration ttl)
       : _sCache = cache
             .withPrefix('$runtimeVersion/$prefix/')
-            .withLogger(logger, ignoreErrors: true)
             .withTTL(ttl)
             .withCodec(utf8),
-        _bCache = cache
-            .withPrefix('$runtimeVersion/$prefix/')
-            .withLogger(logger, ignoreErrors: true)
-            .withTTL(ttl);
+        _bCache = cache.withPrefix('$runtimeVersion/$prefix/').withTTL(ttl);
```

Comment:
`.withLogger`?

Reply:
It disappeared :)

`neat_cache` always ignores errors now... and just returns `null` if `get()` fails.